lib/from_erlang.ml

```diff
@@ -196,7 +210,7 @@ let rec expr_of_erlang_exprs = function
      (* no match expression in `e` by extract_match_expr *)
      let e' = expr_of_erlang_expr' e in
      let es' = expr_of_erlang_exprs es in
-     Case (e', [((pattern_of_erlang_pattern p, Constant (Atom "true")), es')])
+     Case (e', [((pattern_of_erlang_pattern p, Constant (-1, Atom "true")), es')])
```

Comment:
Is there any reason why you used `-1` line number instead of using `F.ExprMatch`'s line number?
I think it is useful to specify line number for debugging even if `Constant` is not appeared in original code.

Reply:
When I wrote that code, `F.ExprMatch`'s line number cannot be used.  Now,  I use its line number.